CHANGES.md

```diff
@@ -7,6 +7,7 @@
   with observe_dask_progress('Writing dataset', 100):
       dataset.to_zarr(store)  
   ```
+* Normalisation may be slow and consume much memory. (#392)
```

Comment:
That's just the symptom and not the solution. Please be clear about what has changed in xcube, so users can undestand what's new in the upcoming release.